pkg/cloudprovider/OWNERS

```diff
@@ -3,7 +3,6 @@
 approvers:
   - mikedanese
   - dims
-  - wlan0
```

Comment:
same comment as https://github.com/kubernetes/kubernetes/pull/106436#discussion_r769719622 ... should we consider kubernetes-sigs activity as well?

Reply:
reverting this change. thanks @liggitt